### PR TITLE
Librtmp new

### PR DESCRIPTION
**Pull request by mathieu-aubin**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -359,6 +359,21 @@ RUN \
...
 
+# bump: librtmp /LIBRTMP_COMMIT=([[:xdigit:]]+)/ gitrefs:https://git.ffmpeg.org/rtmpdump.git|re:#^refs/heads/master$#|@commit
+# bump: librtmp after ./hashupdate Dockerfile LIBRTMP $LATEST
+# bump: librtmp link "Commit diff $CURRENT..$LATEST" https://git.ffmpeg.org/gitweb/rtmpdump.git/commitdiff/$LATEST?ds=sidebyside
+ARG LIBRTMP_URL="https://git.ffmpeg.org/rtmpdump.git"
+ARG LIBRTMP_COMMIT=f1b83c10d8beb43fcc70a6e88cf4325499f25857
+RUN \
+  git clone "$LIBRTMP_URL" && \
+  cd rtmpdump && git checkout $LIBRTMP_COMMIT && \
+  # Patch/port librtmp to openssl 1.1
```
Comment (wader) [inline Dockerfile:370]: I noticed that the ffmpeg package for alpine builds with gnutls, maybe that could be an option? but also noticed that gnutls for alpine does not have any gnutls* package with .a files hmm

File: Dockerfile [hunk 2]
```diff
@@ -359,6 +359,21 @@ RUN \
...
+  for _dlp in dh.h handshake.h hashswf.c; do \
+    wget $WGET_OPTS https://raw.githubusercontent.com/microsoft/vcpkg/master/ports/librtmp/${_dlp%.*}.patch; \
```
Comment (wader) [inline Dockerfile:372]: Could we have a commit hash instead of master in the URL to make things stabler?

**Comment by wader** [comment]
See comments, but overall looks good i think 👍

File: Dockerfile (hunk 2 above)
Comment (mathieu-aubin) [reply Dockerfile:372]: maybe have a patch folder within the project?

Comment (mathieu-aubin) [reply Dockerfile:372]: i dont have much time but... you could also have a patcjed forked on your name, since librtmp hasnt been updated in ages

Comment (mathieu-aubin) [reply Dockerfile:372]: feel free to use this code and create your own with it

**Comment by wader** [comment]
Did #202 that is the same but with commit hash

**Pull request closed without merging**